### ISSUE TITLE
Unable to define campaign in BranchLinkProperties

### ISSUE THREAD

**Issue by mathatan** [RodrigoSMarques/flutter_branch_sdk#29]
`campaign` value is missing from `BranchLinkProperties` constructor parameters in `link_properties.dart`

**Comment by RodrigoSMarques** [comment]
Hi @mathatan 

I checked and the builder doesn't really have the campaign parameter.
However, since the parameters are not final, you can assign the value after the object is created using:

```dart
    BranchLinkProperties lp = BranchLinkProperties(
        channel: 'facebook',
        feature: 'sharing',
        stage: 'new share',
        tags: ['one', 'two', 'three']);

    lp.campaign = 'xxxxxx';
```

I made the correction and soon I will make a new version available with the correction.

Thank you.

**Comment by mathatan** [comment]
Sure thing, thanks!

**Comment by RodrigoSMarques** [comment]
Resolved in new version 0.5.0
